pages/api/book/event.ts

```diff
@@ -265,7 +265,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   const evt: CalendarEvent = {
     type: eventType.title,
-    title: getEventName(reqBody.name, eventType.title, eventType.eventName),
+    title: getEventName(reqBody.name, eventType.title, eventType.eventName, users[0].name || "Nameless"),
```

Comment:
what's Nameless?

Reply:
I think it is a fallback, just in case the name is not found (unlikely, but it is there in the previous bit of code, so I kept it that way to be safe)

Reply:
Can you update this to pass an object instead of 4 arguments?